### PR TITLE
fix: issue 6219

### LINKED ISSUE #6219: Crash: memory access out of bounds
### Describe the bug

RuntimeError: memory access out of bounds

### Input code

```typescript
const enum a{
    a=a
}
```

### Config

```json
{
  "jsc": {
    "parser": {
      "syntax": "typescript",
      "tsx": false
    },
    "target": "es2022",
    "loose": false,
    "minify": {
      "compress": false,
      "mangle": false
    }
  },
  "module": {
    "type": "es6"
  },
  "minify": false,
  "isModule": true
}
```

### Playground link

https://play.swc.rs/?version=1.3.9&code=H4sIAAAAAAAAA0vOzysuUUjNK81VSKzm5VIAgkTbRF6uWgDGSEh5GQAAAA%3D%3D&config=H4sIAAAAAAAAA0WNSwrDMAxE76J1FsWLLnKHHsK4SnDxD40CDcZ3rx1cspNmnp4qfeBorVSsgGVMOJPaL62kZ2E48UVpIUWPNhvArS9WdtaOMMzDmF6HnMETWCj65LdzyFyORRi4K5v28Cdbd8X8PkZQr3%2BX80ntdsw7j9cEVQ5uPx%2Bqcl%2B4AAAA

### Expected behavior

Not crash, but report error

### Actual behavior

Crash

### Version

1.3.9

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by xin2017338**
**Description:** fix [issue 6219](https://github.com/swc-project/swc/issues/6219)

**BREAKING CHANGE:**

**Related issue (if exists):** [issue 6219](https://github.com/swc-project/swc/issues/6219)

### CODE REVIEW

File: crates/swc_ecma_transforms_typescript/src/strip.rs [hunk 1]
```diff
@@ -594,12 +594,6 @@ where
                         if let Some(Some(v)) = values.get(&id.sym) {
                             return Ok(v.clone());
                         }
-                        //
```
Comment (kdy1) [inline crates/swc_ecma_transforms_typescript/src/strip.rs:597]: cc @magic-akari Do you think this is fine?

Comment (magic-akari) [reply crates/swc_ecma_transforms_typescript/src/strip.rs:597]: All test passed. It's OK in my opinion.

**Comment by kdy1** [comment]
Can you sign the CLA?

**Review by kdy1 — APPROVED** [review]
swc-bump:
 - swc_ecma_transforms_typescript

**Comment by suxin2017** [comment]
> Can you sign the CLA?

Yes, I have

**Pull request merged**